src/main/java/org/sonar/plugins/clojure/sensors/kondo/KondoSensor.java

```diff
@@ -0,0 +1,103 @@
+package org.sonar.plugins.clojure.sensors.kondo;
+
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.fs.TextRange;
+import org.sonar.api.batch.rule.Severity;
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.batch.sensor.issue.NewIssue;
+import org.sonar.api.batch.sensor.issue.NewIssueLocation;
+import org.sonar.api.rule.RuleKey;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.plugins.clojure.language.Clojure;
+import org.sonar.plugins.clojure.rules.ClojureLintRulesDefinition;
+import org.sonar.plugins.clojure.sensors.AbstractSensor;
+import org.sonar.plugins.clojure.sensors.CommandRunner;
+import org.sonar.plugins.clojure.sensors.CommandStreamConsumer;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.sonar.plugins.clojure.settings.KondoProperties.DISABLED_PROPERTY;
+import static org.sonar.plugins.clojure.settings.KondoProperties.DISABLED_PROPERTY_DEFAULT;
+import static org.sonar.plugins.clojure.settings.Properties.SENSORS_TIMEOUT_PROPERTY;
+import static org.sonar.plugins.clojure.settings.Properties.SENSORS_TIMEOUT_PROPERTY_DEFAULT;
+
+public class KondoSensor extends AbstractSensor implements Sensor {
+
+    private static final Logger LOG = Loggers.get(KondoSensor.class);
+
+    private static final String PLUGIN_NAME = "clj-kondo";
+    private static final String[] COMMAND =
+            {"with-profile", "analysis", "run", "-m", "clj-kondo.main", "--lint", "src", "--config", "{:output {:format :edn}}"};
```

Comment:
I am not familiar with Kondo, but I believe we need to think a little more about this command. Are these parameters always used in Kondo? What if the user has the source in a different directory? I think we can try to generalize a little more so it could work for more situations. Does it make sense?

Reply:
Thank you for your review! You right, this one need to improve. I will investigate for how to eliminate such hard code.

Reply:
@fsantiag Hi. Sorry for such long delay. I have resolved all comments. Would you mind to review it again?